### PR TITLE
CI for variant calling via Github Actions and CIEVaD

### PR DESCRIPTION
**Pull request by Krannich479**
- test CovPipe's variant calling via CIEVaD
- <s> add shield to the README for indels and SNV F1-score for test status </s>

### CODE REVIEW

**Comment by Krannich479** [comment]
Please suspend the PR at least for the time of CIEVaD being a private repo. We'd need to update tokens in rki-mf1/covpipe2 to make the checks work but this becomes obsolete once the repo goes public.

**Comment by Krannich479** [comment]
In future versions of this _Variant Calling test_ I might add caching such that the benchmark scores are read dynamically from previous runs. At the moment they are static integers. However, for the caching the test needs to be merged into the original CovPipe2 repository first. Plus, I am not certain yet whether this (complexity of system engineering) reduces the understanding of how to use CIEVaD's results. _tbd_

File: README.md [hunk 1]
```diff
@@ -7,7 +7,8 @@
 [![run with docker](https://img.shields.io/badge/run%20with-docker-0db7ed?labelColor=000000&logo=docker)](https://www.docker.com/)
 [![run with singularity](https://img.shields.io/badge/run%20with-singularity-1d355c.svg?labelColor=000000)](https://sylabs.io/docs/)
 ![](https://img.shields.io/badge/licence-GPL--3.0-lightgrey.svg)
-[![](https://img.shields.io/badge/awaiting%20peer%20review-F1000Research-ef8336.svg)](https://doi.org/10.12688/f1000research.136683.1)
+[![](https://img.shields.io/badge/manuscript-F1000Research-ef8336.svg)](https://doi.org/10.12688/f1000research.136683.1)
```
Comment (MarieLataretu) [inline README.md:10]:
```suggestion
[![](https://img.shields.io/badge/manuscript-F1000Research-ef8336.svg)](https://doi.org/10.12688/f1000research.136683.2)
```

File: .github/workflows/VariantCalling.yml
Comment (MarieLataretu) [inline .github/workflows/VariantCalling.yml:1]: Would it make sense to upload result files from CIEVaD as artifacts? 🤔 

I saw it at the lining CI at nf-core:
https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/.github/workflows/linting.yml#L60-L68

and I have it here to check results, when the test failed:
https://github.com/rki-mf1/CoVpipe2/blob/main/.github/workflows/pytest_workflows.yml#L58-L68

**Review by MarieLataretu — APPROVED** [review]

**Pull request merged**